.github/CONTRIBUTING.md

```diff
@@ -16,8 +16,6 @@ All third-party contributions to this project must be accompanied by a signed co
 Magento Documentation works with Magento Community Engineering teams and projects.
 As you contribute PRs, you gain [Contribution Points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#points).
```

Comment:
I just noticed that Contributor Guide is versioned. I don't think there is a reason for that. I want to make it versionless.

Reply:
These files are not part of the published guide, so I would be OK with that.

Reply:
We can update when you change that.

Reply:
magento/devdocs#7591